Packs/AWS-Enrichment-Remediation/Scripts/AWSRecreateSG/AWSRecreateSG.py

```diff
@@ -110,14 +113,21 @@ def sg_fix(sg_info: List, port: int, protocol: str) -> Dict:
             recreate_list.append(str(priv).replace("'", "\""))
         new_name = info['GroupName'] + "_xpanse_ar_" + str(randint(100, 999))
         description = "copied from rule " + info['GroupName'] + " by Xpanse Active Response module"
-        new_sg = demisto.executeCommand("aws-ec2-create-security-group",
-                                        {"groupName": new_name, "vpcId": info['VpcId'], "description": description})
+        cmd_args = {"groupName": new_name, "vpcId": info['VpcId'], "description": description, "using": integration_to_use}
+        if assume_role:
+            cmd_args.update({'roleArn': assume_role})
+            cmd_args.update({'roleSessionName': "test"})
```

Comment:
So, the value of `roleSessionName` is going to always be`test`?
Is that not a confusing name?
Also, wouldn't that cause a conflict if there are multiple sessions running at the same time, and they're all named `test`?

(this note is true for other places in the PR where the `roleSessionName` is used)

Reply:
I have tested and it wasn't an issue, I will name it something else that should be a bit better

Reply:
change in [review updates](https://github.com/demisto/content/pull/29311/commits/d0b01a94de4508529522275932721c124c094d62)

Reply:
Let's set a constant at the top, something like `ROLE_SESSION_NAME = "xsoar-session"`, and use that as the value.
This way we ever decide to change that for some reason, we change it in one place instead of multiple places.